code/client/index.html

```diff
@@ -112,7 +112,7 @@
 	<link href="/vendors/owl-carousel/owl.carousel.css" rel="stylesheet" type="text/css" />
 	<link href="/vendors/owl-carousel/owl.theme.css" rel="stylesheet" type="text/css" />
 </head>
-<body onload="ApplyDesignExperiemnt();">
+<body class="design-v2" onload="ApplyDesignExperiemnt();">
```

Comment:
This has been merged. Which will always show new design in the experiment. Let me submit a new PR with this removed